src/v3/src/util/locUtil.ts

```diff
@@ -17,12 +17,13 @@ export const loc = (
   key: string,
   bundleName?: string,
   params?: Array<string | number | boolean | unknown>,
+  replacerFn?: (content: string) => string,
 ): string => {
-  const localizedText = localize(key, bundleName, params);
+  const localizedText: string = localize(key, bundleName, params);
 
-  return localizedText
-    .replace('<$1>', '')
-    .replace('</$1>', '')
-    .replace('<span class="strong">', '')
-    .replace('</span>', '');
+  if (typeof replacerFn === 'function') {
+    return replacerFn(localizedText);
```

Comment:
Rather than allow passing an unrestricted function, what do you think of passing a map of key/value replacements? Right now the function can modify the string in any way it wishes.